### PR TITLE
Vaccine Tracking example

### PR DESCRIPTION
**Pull request by j4ys0n**
This adds a demo example for tracking logistics around a hypothetical vaccine manufacturing scenario.

### CODE REVIEW

**Review by atoulme — APPROVED** [review]
looks good to me

File: examples/vaccine-demo/chaincode/high-throughput-splunk.go [hunk 1]
```diff
@@ -0,0 +1,603 @@
...
+)
+
+//SmartContract is the data structure which represents this contract and on which  various contract lifecycle functions are attached
+type SmartContract struct {
+}
+
+type item struct {
+	object_type string `json:"docType"`
+	item_uid string `json:"item_uid"`
+  item_name string `json:"item_name"`
```
Comment (ziegfried) [inline examples/vaccine-demo/chaincode/high-throughput-splunk.go:43]: nit: gofmt please

File: examples/vaccine-demo/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Splunk for Hyperledger Fabric - Vaccine Tracking Demo
+
+This is a demo environment to show what's possible with the integration between Splunk and Hyperledger Fabric. To start the environment, first ensure that you have Docker installed.
+
+https://docs.docker.com/get-docker/
+
```
Comment (ccordi) [inline examples/vaccine-demo/README.md:6]:
```suggestion

Next, download the Hyperledger Fabric samples, binaries, and Docker images.
```

**Review by ccordi — COMMENTED** [review]
Adding a comment describing the curl request would be good.

File: examples/vaccine-demo/chaincode/high-throughput-splunk.go (hunk 1 above)
Comment (j4ys0n) [reply examples/vaccine-demo/chaincode/high-throughput-splunk.go]: done

**Pull request merged**